Corona-Warn-App/src/main/java/de/rki/coronawarnapp/submission/DefaultKeyConverter.kt

```diff
@@ -6,16 +6,20 @@ import com.google.protobuf.ByteString
 
 class DefaultKeyConverter : KeyConverter {
 
-    override fun toExternalFormat(key: TemporaryExposureKey, riskValue: Int) =
+    companion object {
+        private const val ROLLING_PERIOD = 144
+    }
+
+    override fun toExternalFormat(
+        key: TemporaryExposureKey,
+        riskValue: Int,
+        daysSinceOnsetOfSymptoms: Int
+    ) =
         KeyExportFormat.TemporaryExposureKey.newBuilder()
             .setKeyData(ByteString.readFrom(key.keyData.inputStream()))
             .setRollingStartIntervalNumber(key.rollingStartIntervalNumber)
             .setRollingPeriod(ROLLING_PERIOD)
```

Comment:
Why are we explicitly setting the rolling period here?
@jakobmoellersap do you know? After all, it was changed as part of the PR but I'm wondering whether it should be...

Reply:
Afaik, this was previously set to 144 by default. WIth 15th Key this should not be defaulted but taken from the key info.

Reply:
thank you

Reply:
@jakobmoellersap thanks, that's also what I was thinking...